src/views/project/ProjectList.vue

```diff
@@ -1,23 +1,87 @@
-<template>
-  <div class="main">
-    <Main />
-  </div>
+<template lang="pug">
+  div(class="main")
+    div
+      v-row
+        v-col(md="6")
+          h2(class="display-1") Projects
+        v-col(md="6")
+          div(class="float-right")
+            CreateUpdateProject(v-bind:onFinish="onFinish")
+      ErrorIndicator(v-if="error") Error while loading projects!
+      div(v-else)
+        Loading(v-if="loading")
+        v-list(v-else color="transparent")
+          template(v-if="!projects.length")
+            br
+            div(class="text-center")
+              v-icon(x-large) mdi-folder
+              br
+              div No projects found!
+          template(v-else)
+            v-list-item(v-for="project in projects" :key="project.id" class="project-item" v-bind:href="`/project/${project.id}`")
+              v-card(width="100%" class="project-item-card" elevat)
+                v-card-title {{project.name}}
+                v-card-text
+                  div(class="text--primary") {{project.description}}
+                  | Created by 
+                  strong "user"
+                  | on {{moment(project.created).format('lll')}}
+                  div(class="teal--text") 
+                    strong {{project.texts.length}} texts, 0 relations
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
-import Main from '@/components/project/Main.vue';
+import { Component, Vue } from 'vue-property-decorator';
+import { AxiosResponse } from 'axios';
+import { Location } from 'vue-router';
 
-export default Vue.extend({
-  name: 'project',
+import CreateUpdateProject from '@/components/project/CreateUpdateProject.vue';
+import Loading from '@/components/global/Loading.vue';
+import ErrorIndicator from '@/components/global/ErrorIndicator.vue';
+import { Project, Text } from '@/components/project/models';
+import { VForm, PaginatedResult } from '@/models';
+
+@Component({
+  name: 'ProjectList',
   components: {
-    Main,
+    Loading,
+    ErrorIndicator,
+    CreateUpdateProject,
   },
-});
+})
+export default class ProjectList extends Vue {
+  private projects: Project[] = [];
+  private loading: boolean = true;
+  private error: boolean = false;
+
+  public mounted(): void {
+    this.getProjectList();
+  }
+
+  private getProjectList() {
+    Vue.$axios.get('/project')
+      .then((response: AxiosResponse) => {
+        const data: PaginatedResult<Project> = response.data as PaginatedResult<Project>;
```

Comment:
Why do you save the response into data then save data into projects? Why not just save what you need directly into projects without creating an intermediate variable. I do not see data being used anywhere else